How We Work updates

ssatagopan opened the pull request:
# [Sprint Planning update](https://github.com/orgs/OHS-Hosting-Infrastructure/projects/1#card-63184494)

Changes proposed in this pull request:

- Added Conventions section
- Updated Sprint Planning section

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,6 +19,26 @@ Important points:
+   * 2: Well-understood and can be completed with little to no interaction between teams in a few days
+   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion
+ * Point estimates are put in parenthesis at end of ticket title
+ * Epics or high level initiative name is put in square brackets at beginning of ticket title
```
```suggestion
  * All team members vote on points, whether or not they're in a technical role. 
  * Epics or high level initiative name is put in square brackets at beginning of ticket title
```

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,6 +19,26 @@ Important points:
+   * etc.
+ * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+   * If any item is estimated as 5, needs to be broken down
+   * 1: Well-understood and can be completed without requiring interaction between teams within 1 day
```
```suggestion
   * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours
```

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,6 +19,26 @@ Important points:
+ * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+   * If any item is estimated as 5, needs to be broken down
+   * 1: Well-understood and can be completed without requiring interaction between teams within 1 day
+   * 2: Well-understood and can be completed with little to no interaction between teams in a few days
```
```suggestion
   * 2: Well-understood and can be completed with little to no interaction between teams within half a day of concentrated work
```

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,6 +19,26 @@ Important points:
+   * If any item is estimated as 5, needs to be broken down
+   * 1: Well-understood and can be completed without requiring interaction between teams within 1 day
+   * 2: Well-understood and can be completed with little to no interaction between teams in a few days
+   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion
```
```suggestion
   * 3: Some ambiguity or well understood but requires interaction with other teams that may delay completion, likely would take 1 full work day
```

kilbergr reviewed (changes requested):
So I think your additions are ab fab. However, if this is for the ticket about updating our sprint process, I think there's more to be added. One thing that stands out to me is there's no mention of grooming. Another is that we are doing 1 week sprints.

ssatagopan commented:
> So I think your additions are ab fab. However, if this is for the ticket about updating our sprint process, I think there's more to be added. One thing that stands out to me is there's no mention of grooming. Another is that we are doing 1 week sprints.

Added Sprint Grooming content, plus adjusted sprint length, start day, and added Definition of Ready as TODO

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,21 +19,55 @@ Important points:
+Grooming activities include:
+* Adjusting placement of tickets in backlog based on priority
+* Ticket clarification ensuring that Acceptance Criteria is clear
+* Estimation of tickets and splitting of tickets if needed
```
What's estimation of tickets? That's not pointing, is it?

ssatagopan replied:
Yes, pointing.

kilbergr replied:
We don't do pointing in grooming.

kilbergr replied:
We COULD split them, though.

ssatagopan replied:
hmm...so we only point during sprint planning? it helps provide visibility for the work in backlog if more tickets than just what's for next sprint are estimated.

kilbergr replied:
I've always done that way. Happy to try out this suggestion, but then we have the issue of everyone having to be at both grooming and sprint planning. Generally I like to keep grooming smaller. What do you think?

ssatagopan replied:
As discussed after standup, it's helpful for full team to be present at both meetings.

kilbergr commented on docs/how-we-work/README.md:
```diff
@@ -19,21 +19,55 @@ Important points:
+ * Ticket estimation:
+   * Items are estimated using Fibonnaci points of 1, 2, 3, and 5
+     * If any item is estimated as 5, needs to be broken down
+     * 1: Well-understood and can be completed without requiring interaction between teams in less 1-2 work hours
```
```suggestion
     * 1: Well-understood and can be completed without requiring interaction between teams in less than 1-2 work hours
```
Sorry that's my b

kilbergr reviewed (approved):
One missing word and a question, otherwise I think looking good!
